integration_tests/backend/test_reads.py

```diff
@@ -347,6 +348,16 @@ def test_endpoint_workflows_get(self):
                     assert key in v2_workflow
                 assert v2_workflow["user_id"] == user_id
 
+    def test_endpoint_workflow_dags_get(self):
+        flow_id, _ = self.flows["flow_with_metrics_and_checks"]
+        resp = self.get_response(self.GET_DAGS_TEMPLATE % flow_id)
+        resp = resp.json()
+
+        assert len(resp) == 2
```

Comment:
To check the format of the response, I cast to a Response class object. I think you can do that for this too. As bonus, it will be easier for me to integrate into the SDK if the Response classes are already written up.